.github/workflows/ci.yml

```diff
@@ -315,6 +315,46 @@ jobs:
           flags: ${{ steps.test.outputs.codecov_flag }}
           fail_ci_if_error: false
 
+  macos-test:
+    needs: prep-testbed
+    runs-on: macos-latest
+    env:
+      JINA_DAEMON_BUILD: DEVEL
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.7]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Prepare enviroment
+        run: |
+          brew install docker-machine docker
+          docker-machine create default
+          docker-machine env default
+          eval $(docker-machine env default)
+          brew services start docker-machine
+          docker build -f Dockerfiles/pip.Dockerfile -t jinaai/jina:test-pip .
```

Comment:
So this just tests that the image is built, right? It is not used for testing?

Reply:
Sorry, this PR is not ready yet.

Reply:
Oh ok